docs/proposals/custom-playbook-names.md

```diff
@@ -0,0 +1,58 @@
+# Custom Playbook Names
+
+## Introduction
+Instead of forcing the user to have a playbook named provision, deprovision,
+bind, or unbind, allow the user to create a single playbook and select the
```

Comment:
I know that this was the direction that I was headed with the APB reference example. The only difference between the different playbooks was the `action` variable (provision, deprovision, ...). If you have a look at https://github.com/ansibleplaybookbundle/hello-world-apb/pull/3 you will see that the APB is structured as a single ansible role installed as `apb` in the resulting APB image. Not sure how much that factors into this discussion, but I think it is something you could consider as you develop this proposal.

Reply:
I want to use a single playbook with a variable to reflect the action.  Which is the same structure as your PR.

Reply:
This could be handled with a shim playbook that calls the existing role without modification, true?  Also, help me understand more about why one would would want multiple APBs in the same repo?